dora-metrics/pelorus
# Allow skopeo to use certificate from the mounted to pod location

Adds option to skopeo that allows to use mounted ca.crt. Adds more informative debug information for the skopeo committime exporter Adds system:image-puller to the pelorus-exporter Service Account


## Testing Instructions

Use built operator version for the skopeo exporter.

Ensure `pelorus-exporter` service account has the corresponding RoleBinding `pelorus-exporters-image-puller` with the `system:image-puller` reference in the namespace where it was installed (default `pelorus`).

Enable exporter with the debug information to see if the debug was printed to the console.
